extensions/toggl-track/src/menuBar.tsx

```diff
@@ -74,8 +66,9 @@ export default function Command() {
         <MenuBarExtra.Section title="Running time entry">
           <MenuBarExtra.Item
             icon={{ source: Icon.Circle, tintColor: runningEntry?.project_color }}
-            onAction={() => {
-              stopRunningTimeEntry(runningEntry);
+            onAction={async () => {
+              await stopRunningTimeEntry(runningEntry);
+              await sleep(250);
             }}
```

Comment:
This is super weird, but I'm glad you found a workaround. Make sure you place a comment on that line explaining the reason for the workaround. Same for the other line as well.

Reply:
Been doing a lot of talking on Raycast's Slack. This is what was the final solution offer. It's also used in 
https://github.com/raycast/extensions/blob/main/extensions/acqua/src/menubar.tsx#L11-L14

Good call on the comment, will add